docker-compose.yml

```diff
@@ -1,31 +1,44 @@
 services:
   tailscale-app:
-    image: tailscale/tailscale:latest
+    image: tailscale/tailscale:latest # Use the latest Tailscale image
     hostname: couch-tailscale
+    container_name: tailscale-app # Proper name for the Tailscale container
     env_file: .env.tailscale-app
     volumes:
       - tailscale-data:/var/lib/tailscale
       - ${PWD}/ts-conf/ts-serve.json:/config/ts-serve.json:ro
       - /dev/net/tun:/dev/net/tun
+      - /lib/modules:/lib/modules:ro # Crucial for modprobe and kernel module access
     cap_add:
-      - net_admin
-      - sys_module
+      - NET_ADMIN
+      - SYS_MODULE # Crucial for modprobe
+      - NET_RAW # Often helpful for network manipulation
+      - NET_BIND_SERVICE
+    networks: 
+      - funnel_network
     restart: unless-stopped
-  app:
-    image: couchdb:3.4.2
-    network_mode: service:tailscale-app
+
+  couchdb-app:
+    image: couchdb:latest # Use the latest CouchDB image
```

Comment:
CouchDB is extremely stable, but there are very rare breaking changes. This is where strategies may differ, but it may be better to pin it as minor. I am not familiar with the docker-compose, but can we pin to 3.4.x? (Of course, 3.5.0 has been released, but I have not checked yet...)